catalog/task/collect-extra-data/0.2/collect-extra-data.yaml

```diff
@@ -0,0 +1,101 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: collect-extra-data
```

Comment:
I wonder if we should rename this to something like `collect-release-resources` as it does so much more than just fetching `extraData` now. WDYT?

Reply:
I like this. @davidmogar ?

Reply:
Yeah, it's basically one of the things I was discussing with you offline. That why I wanted to wait on the outcome of our discussion before making further changes. Another name could be `collect-data` if we go in that direccion.

Reply:
what about `collect-release-data` ?

Reply:
I think this is the last thing unresolved. @davidmogar are you good with me renaming it? I guess to `collect-data`?

Reply:
Well, I don't want to hold up this PR on if we rename the field as we discussed yesterday. So which one would you rather considering the other decision hasn't been made yet?

Reply:
`collect-data` sounds good to me.